beater/common_handlers.go

```diff
@@ -304,19 +302,15 @@ func killSwitchHandler(killSwitch bool, h http.Handler) http.Handler {
 }
 
 const (
-	rateLimitCacheSize       = 1000
-	rateLimitBurstMultiplier = 2
+	rateLimitCacheSize = 1000
 )
 
 func ipRateLimitHandler(rateLimit int, h http.Handler) http.Handler {
-	cache, _ := lru.New(rateLimitCacheSize)
+	rlc, _ := NewRlCache(rateLimitCacheSize, rateLimit, rateLimitBurstMultiplier)
```

Comment:
Why did you remove the `rateLimitBurstMultiplier = 2` const and where does `rateLimitBurstMultiplier` now come from?

Reply:
sorry, that was a bad commit - fixed now